docker-compose.yml

```diff
@@ -10,6 +10,10 @@ services:
             - db_data:/var/lib/postgresql/data
         networks:
             - backend
+        environment:
```

Comment:
I don't think the database needs any of these environment variables.

Reply:
Those were the three I added to make it work. I'm not sure if the first two are necessary. I will test it without them and add a new commit if they are not needed.

Reply:
Sorry, my bad, reading too fast! `POSTGRES_HOST_AUTH_METHOD` here, but not below. The other two below but not here?